fastlane/metadata/android/en-US/changelogs/983.txt

```diff
@@ -0,0 +1,9 @@
+New fast forward / rewind behaviour
+Made settings searchable
+Highlight pinned comments as such
+Added FSFE PeerTube instance to URL opener
+Improved error reporting and new error notification
+Fixed first item always played in the play queue when reloading play queue manager
```

Comment:
Probably most of users don't know what is the play queue manager and where/when/how it is used so I'd rename this entry to:

```suggestion
Fixed first item always played in the play queue when switching quality or player type
```